app/src/main/java/app/gamenative/ui/screen/xserver/XServerScreen.kt

```diff
@@ -1678,6 +1683,24 @@ private fun extractx86_64InputDlls(context: Context, container: Container) {
     } else Log.d("XServerDisplayActivity", "Wine version is not proton-9.0-x86_64, skipping input dlls extraction")
 }
 
+private fun extractProton10Arm64ecInputDLLs(context: Context, container: Container) {
+    val inputAsset = "proton10_arm64ec_input_dlls.tzst"
+    val imageFs = ImageFs.find(context)
+    val wineVersion: String? = container.getWineVersion()
+    Log.d("XServerDisplayActivity", "Proton 10 arm64ec Input DLL Extraction Verification: Container Wine version: " + wineVersion)
+
+    if (wineVersion != null && wineVersion.contains("proton-10.0-arm64ec")) {
+        val wineFolder: File = File(imageFs.getWinePath() + "/lib/wine/")
+        Log.d("XServerDisplayActivity", "Wine version contains proton-10.0-arm64ec. Extracting input dlls to " + wineFolder.getPath())
+        val success: Boolean = TarCompressorUtils.extract(TarCompressorUtils.Type.ZSTD, context.assets, inputAsset, wineFolder)
+        if (!success) {
+            Log.e("XServerDisplayActivity", "Failed to extract Proton 10 arm64ec input dlls")
+        }
+    } else {
+        Log.d("XServerDisplayActivity", "Wine version is not proton-10.0-arm64ec, skipping Proton 10 input dlls extraction.")
+    }
+}
+
```

Comment:
This is something that I wrote due to the input DLLs being broken on the Proton-10 I tested on. 

This is technically not needed because there's now a fixed version of that proton-10 dist.